src/pdn/src/pdn.tcl

```diff
@@ -635,6 +635,177 @@ proc add_pdn_connect {args} {
                     $dont_use
 }
 
+
+sta::define_cmd_args "add_sroute_connect" {
```

Comment:
Please give some documentation for the command and its arguments.  Please add a unit test.